sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-preview.4 (Unreleased)
 
 
 ## 1.0.0-beta.1 (2020-11-22)
```

Comment:
After the next version bump from our automation this beta will sort weirdly. I suggest either now or then we just fold these notes into the new preview version. Also to help clean this up on nuget I unlisted the beta.1 version from there.

Reply:
Will do in 2nd PR.